cmd/kola/options.go

```diff
@@ -62,6 +61,7 @@ func init() {
 	sv(&kola.Options.IgnitionVersion, "ignition-version", "", "Ignition version override: v2, v3")
 	ssv(&kola.BlacklistedTests, "blacklist-test", []string{}, "List of tests to blacklist")
 	bv(&kola.Options.SSHOnTestFailure, "ssh-on-test-failure", false, "SSH into a machine when tests fail")
+	sv(&kola.Options.CosaBuild, "cosa-build", "", "File path for coreos-assembler meta.json")
```

Comment:
Would there be any advantage to having the parameters be the top level COSA dir & the build-id over a path to the meta.json for the build?

Reply:
That'd be better, it'd just involve moving a good bit more code, including parsing `builds.json` etc.